scripts/bert/baidu_ernie_data.py

```diff
@@ -0,0 +1,104 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=line-too-long
+"""GLUEBenchmark corpora."""
```

Comment:
Can you also add this to `__init__.py`? It's causing some errors on CI: http://ci.mxnet.io/blue/organizations/jenkins/GluonNLP-py3-master-cpu-unittest/detail/PR-717/8/pipeline

Reply:
changed to """Baidu ernie data, contains XNLI."""

Reply:
Also, baidu_ernie_data import added in __init__.py

Reply:
Fixed the import error by using `from __future__ import absolute_import`, and use explicit relative import. It looks like that the error was cause by implicit relative import in py3.

Reply:
It looks like CI still cannot import the package for other scripts. Can we add this to init.py?